spikeextractors/extractors/nwbextractors/nwbextractors.py

```diff
@@ -517,12 +492,11 @@ def add_electrodes(recording: se.RecordingExtractor, nwbfile=None,
                           " the nwbfile electrode ids and RecordingExtractor channel ids!")
 
         for j, channel_id in enumerate(channel_ids):
-            # Will not overwrite the electrode id if already in nwbfile
             if channel_id not in nwb_elec_ids:
-                electrode_kwargs = fill_kwargs_from_defaults(defaults)
+                electrode_kwargs = dict(defaults)
```

Comment:
why use `dict` here?

Reply:
To make a copy (or a new dictionary object) filled with default values, in case operations (like `.pop()` or `.update()`) need to be performed on it for some reason afterward.

Reply:
ah, it's used as a copy. Makes sense, thanks for the explanation :-)

Reply:
Np, it's really only necessary at like one or two parts of the code that actually arise as byproducts of the removal of the previous `fill_kwargs_from_defaults` function, which is actually subtly different from `dict(defaults, **values)` in that it only looped over keys in defaults, so extra keys in values would not actually end up in the final `kwargs`, so in the new method it just has to be popped off before going into the `add_*` function (see line 373 for one of the only occurrences; it may even disappear as I refactor the other functions to match the new style).

Thanks again for all the suggestions! I'll try to get this done by tomorrow-ish pending the progress of the upload speed on that other DANDI set.